lib/langchain/llm/aws_bedrock.rb

```diff
@@ -260,5 +281,29 @@ def compose_parameters_ai21(params)
         }
       }
     end
+
+    def response_from_chunks(chunks)
```

Comment:
Do you think you could add a unit test just this method?

Reply:
sure!